src/vendor/mod.rs

```diff
@@ -5,6 +5,4 @@
 //! them. Note that we separate them by vendor-name, which is not the best
 //! name-space but should be acceptible.
 
-pub mod intel {
-    pub mod console_control;
-}
+pub mod intel;
```

Comment:
Can you explain the rationale behind `build: fix project structure`? What is the advantage over the existing model? Is this driven by some style guideline? I just wonder what triggered you to change it, or why you prefer one over the other.

Reply:
Yeah, I understand. That's why I expect contributors to do this conversion whenever they happen to add more sub-modules to those respective modules. But until then, I always feel like it makes it much harder to get good overviews of r-efi if every single module is moved into its own respective subdirectory.

Reply:
@dvdhrm Ok, we can leave it be until then.